modules/ztp-pre-staging-downloading-artifacts.adoc

```diff
@@ -0,0 +1,367 @@
+----
+
+. To access the container registry, copy a valid pull secret on the server to be installed:
+
+.. Create the `.docker` folder:
++
+[source,terminal]
+----
+$ mkdir /root/.docker
+----
+
+.. Copy the `config.json` file to the previously created `.docker` folder:
++
+[source,terminal]
+----
+$ cp config.json /root/.docker/config.json <1>
+----
+<1> `/root/.docker/config.json` is the default path where `podman` checks for the login credentials for the registry.
+
+[NOTE]
+====
+If you use a different registry to pull the required artifacts, you need to copy the proper pull secret.
+If the local registry uses TLS, you need to include the certificates from the registry as well.
+====
+
+[id="ztp-downloading-ocp-images_{context}"]
+== Downloading the {product-title} images
+
+The {factory-prestaging-tool} allows you to precache all the container images required to provision a specific {product-title} release.
+
+.Procedure
+
+. Pre-cache the release by running the following command:
++
+[source,terminal]
+----
+# podman run -v /mnt:/mnt -v /root/.docker:/root/.docker --privileged --rm quay.io/openshift-kni/telco-ran-tools -- \
+   factory-precaching-cli download -r 4.11.5 -f /mnt \
+   --ai-img registry.redhat.io/multicluster-engine/assisted-installer-agent-rhel8@sha256:da1753f9fcb9e229d0a68de03fac90d15023e647a8db531ae489eb93845d5306 \
+   --ai-img registry.redhat.io/multicluster-engine/assisted-installer-reporter-rhel8@sha256:e8d6b78248352b1a8e05a22308185a468d4a139682d997a7f968b329abbc02cd \
+   --ai-img registry.redhat.io/multicluster-engine/assisted-installer-rhel8@sha256:33abd6e21cfdc36dd4337fa6f3c3442d33fc3f976471614dca5b8ef749e7a027 \
+   --img quay.io/user/troubleshoot
+----
+
++
+.Example output
+[source,terminal]
+----
+Generated /mnt/imageset.yaml
+Generating list of pre-cached artifacts...
+Processing artifact [1/176]: ocp-v4.0-art-dev@sha256_6ac2b96bf4899c01a87366fd0feae9f57b1b61878e3b5823da0c3f34f707fbf5
+Processing artifact [2/176]: ocp-v4.0-art-dev@sha256_f48b68d5960ba903a0d018a10544ae08db5802e21c2fa5615a14fc58b1c1657c
+Processing artifact [3/176]: ocp-v4.0-art-dev@sha256_a480390e91b1c07e10091c3da2257180654f6b2a735a4ad4c3b69dbdb77bbc06
+Processing artifact [4/176]: ocp-v4.0-art-dev@sha256_ecc5d8dbd77e326dba6594ff8c2d091eefbc4d90c963a9a85b0b2f0e6155f995
+Processing artifact [5/176]: ocp-v4.0-art-dev@sha256_274b6d561558a2f54db08ea96df9892315bb773fc203b1dbcea418d20f4c7ad1
+Processing artifact [6/176]: ocp-v4.0-art-dev@sha256_e142bf5020f5ca0d1bdda0026bf97f89b72d21a97c9cc2dc71bf85050e822bbf
+...
+Processing artifact [175/176]: ocp-v4.0-art-dev@sha256_16cd7eda26f0fb0fc965a589e1e96ff8577e560fcd14f06b5fda1643036ed6c8
+Processing artifact [176/176]: ocp-v4.0-art-dev@sha256_cf4d862b4a4170d4f611b39d06c31c97658e309724f9788e155999ae51e7188f
+----
+
+.Verification
+
+. Check that all the images are compressed in the target folder of the bare-metal server:
```

Comment:
why bare-metal? I know it is mainly targeting bare-metal servers in telco, but we did not mention that bare-metal server is a prereq. WDYT?

Reply:
I'm not sure where bare-metal came from, so I removed it. It just says "...folder of the server." to not cause any confusion.